.github/workflows/ci_tests.yml

```diff
@@ -10,11 +10,16 @@ env:
   GEOSX_TPL_TAG: 238-63
 
 jobs:
+
   # Matrix jobs will be cancelled if PR is a draft.
   # PR status must be "Open" to run CI.
   check_pull_request_is_not_a_draft:
     # We use the most recent ubuntu distribution available in Github Actions to ensure maximum support of google cloud's sdk.
     runs-on: ubuntu-22.04
+    # Workaround due to github actions variable scope restrictions passing to
+    # another job's env
+    outputs:
+      GEOSX_TPL_TAG: ${{ env.GEOSX_TPL_TAG}}
```

Comment:
Do you need to remove `GEOSX_TPL_TAG: 238-63` line 10 then?

Reply:
> Do you need to remove GEOSX_TPL_TAG: 238-63 line 10 then?

It is still necessary.
The expression here is to pass `GEOSX_TPL_TAG` to the reusable workflow.